docs/analyzer/developing_recognizers.md

```diff
@@ -1,3 +1,73 @@
 # Recognizers Development - Best Practices and Considerations
 
-TODO: Copy and update from V1
+Recognizers are the main building blocks in Presidio. Each recognizer is in charge of detecting one or more entities in one or more languages.
+Recognizers define the logic for detection, as well as the confidence a prediction receives and a list of words to be used when context is leveraged.
+
+## Implementation Considerations
+
+### Accuracy
+
+Each recognizer, regardless of its complexity, could have false positives and false negatives. When adding new recognizers, we try to balance the effect of each recognizer on the entire system. A recognizer with many false positives would affect the system's usability, while a recognizer with many false negatives might require more work before it can be integrated. For reproducibility purposes, it is be best to note how the recognizer's accuracy was tested, and on which datasets.
+For tools and documentation on evaluating and analyzing recognizers, refer to the [presidio-research Github repository](https://www.github.com/presidio-research).
```

Comment:
This - and the two following links give me a 404 - is this a private repo perhaps?

Reply:
Thanks! fixed the github link. The others are causing issues because the branch name has a / in it. I'll verify this with a different branch.